sycl/include/sycl/ext/oneapi/matrix/matrix-jit.hpp

```diff
@@ -451,6 +451,276 @@ class wi_element {
   }
 };
 
+template <size_t NumRows, size_t NumCols, matrix_layout Layout, typename Group>
+class wi_element<uint16_t, NumRows, NumCols, Layout, Group> {
```

Comment:
I think we should specialize this class on `bfloat16` type.

Reply:
Alexey, we are adding the support of bfloat16 type as the next step in a separate PR.
This PR is to extend the support of the matrix bf16 type represented by unsigned short.
so we are not changing the way we treat bf16 right now.
Instead, we plan on keeping both: unsigned short as bf16, then SYCL bfloat16 type.
We will keep both until the SYCL bfloat16 makes it to the specification (it is experimental right now)

Reply:
@AlexeySotkin @dkhaldi For now we still consider uint16_t as bf16 in matrix and in the future we will support sycl's bf16.